### PR TITLE
[MPM] Change `MPMAnalysis` in `MpmAnalysis`

### PR DESCRIPTION
**Pull request by ncrescenzio**
Change `MPMAnalysis` in `MpmAnalysis`. This change is required otherwise the class cannot not imported when executing `KratosMain.py`

### CODE REVIEW

File: applications/MPMApplication/python_scripts/mpm_analysis.py [hunk 1]
```diff
@@ -10,7 +10,7 @@
 import itertools
 
 
-class MPMAnalysis(AnalysisStage):
```
Comment (rubenzorrilla) [inline applications/MPMApplication/python_scripts/mpm_analysis.py:13]: Note that by doing this you are breaking all the previous cases that directly call the constructor. I'd suggest to leave an empty class on top of the file to properly warn the user. I mean something of this sort
```py
class MPMAnalysis():
    def __init__(self, model, project_parameters):
        raise Exception("MPMAnalysis is now MpmAnalysis bla bla bla")
```
This you can remove it after a prudential deprecation period.

File: applications/MPMApplication/python_scripts/kratos_main_mpm.py [hunk 2]
```diff
@@ -1,10 +1,10 @@
```
Comment (rubenzorrilla) [inline applications/MPMApplication/python_scripts/kratos_main_mpm.py:1]: Note that if you follow the naming convention you can use the standard main file as the other applications do.

Comment (rubenzorrilla) [reply applications/MPMApplication/python_scripts/kratos_main_mpm.py:1]: [MainKratos.zip](https://github.com/user-attachments/files/16314422/MainKratos.zip)
 I mean this one.

File: applications/MPMApplication/python_scripts/mpm_analysis.py (hunk 1 above)
Comment (ncrescenzio) [reply applications/MPMApplication/python_scripts/mpm_analysis.py:13]: Done! Thanks for the suggestion

File: applications/MPMApplication/python_scripts/kratos_main_mpm.py (hunk 2 above)
Comment (ncrescenzio) [reply applications/MPMApplication/python_scripts/kratos_main_mpm.py:1]: This is exactly the reason why I'm changing the name from `MPMAnalysis` to `MpmAnalysis`. So I think I can also remove the script `kratos_main_mpm.py` from the application, right?

**Review by VeronikaSinger — APPROVED** [review]

File: applications/MPMApplication/python_scripts/kratos_main_mpm.py (hunk 2 above)
Comment (ncrescenzio) [reply applications/MPMApplication/python_scripts/kratos_main_mpm.py:1]: sorry @rubenzorrilla, can you confirm me that everything is ok before merging?

Comment (rubenzorrilla) [reply applications/MPMApplication/python_scripts/kratos_main_mpm.py:1]: Everything looks good to me. Thanks for taking my comments into consideration.

**Pull request merged**